molgenis-api-tests/src/test/java/org/molgenis/api/tests/oneclickimporter/OneClickImporterControllerAPIIT.java

```diff
@@ -132,19 +133,9 @@ private void oneClickImportTest(String fileToImport) throws URISyntaxException
 		List<String> validJobStats = Arrays.asList("PENDING", "RUNNING", "SUCCESS");
 		assertTrue(validJobStats.contains(jobStatus));
 
-		// Poll job until it finishes
-		int pollIndex = 0;
-		int maxPolls = 10;
-		long pollInterval = 500L;
-		while ((!jobStatus.equals("SUCCESS")) && pollIndex < maxPolls)
-		{
-			LOG.info("Import job status : " + jobStatus);
-			jobStatus = pollJobForStatus(jobUrl);
-			waitForNMillis(pollInterval);
-			pollIndex++;
-		}
-		LOG.info("Import job status : " + jobStatus);
-		assertEquals(jobStatus, "SUCCESS");
+		await().pollDelay(500, MILLISECONDS)
```

Comment:
Could you write a section in the testing chapter of our developer guidelines containing an example like this?

Reply:
Done!